_posts/2020-11-07-welcome-to-atom.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title:  "Welcome to Atom!"
+date:   2020-11-07 12:54:04 -0700
+categories: Atom markdown GitHub html
+---
+Hello! This blog post will walk you through the basic steps on how to set up and
+use Atom. Atom is an awesome and free text editing tool that allows you to write
+Markdown and convert it into an html.
+# Setting up Atom
+First, let's start by installing Atom onto your computer:
+ 1. Visit the [Atom website](https://atom.io/)
+ 2. Click **Download** and install
+ 3. You will get a bunch of welcome screens, but you can simply close them
+ 4. Next, you will click **File** and then **New File**
+ 5. If you do not want your lines going across the whole screen, you can click
+   **View** and then **Toggle Soft Wrap**
+ 6. Now you are ready to start typing!
+
+### Downloading Packages
+Our next step will be to download the necessary packages before we can get
+to learning how to write cool syntax.
```

Comment:
This is a good transition sentence; however, I would change "get to learning" to "get started on learning".  This will help the sentence flow.